client/pages/details.jsx

```diff
@@ -60,6 +62,10 @@ export default class Details extends React.Component {
       .then(result => {
         const test = this.state.comments.slice();
         test.push(result[0]);
```

Comment:
You should rename this variable `test` is a horseshit variable name, just name it `slicedComments` or something.